Replicator/tests/SGTestUser.cc

```diff
@@ -3,21 +3,39 @@
 //
 
 #include "SGTestUser.hh"
-#include "Defer.hh"
 
 bool SG::TestUser::addChannels(const std::vector<std::string>& channels) {
+    REQUIRE(_sg);
```

Comment:
This is more related to the test framework itself. Perhaps Assert is more apt.

Reply:
Thank you, I was unsure about this